.node-version

```diff
@@ -1 +1 @@
-6.5.0
+8.5.0
```

Comment:
why ? (I've node 6.5.x installed locally to mimic the Azure Function environment)

Reply:
ops removed